internal/commands/pull/command.go

```diff
@@ -48,26 +49,48 @@ func (cmd *Command) Flags() []flags.Flag {
 		flags.StringFlag{
 			Value: &cmd.inputs.RemoteApp,
 			Meta: flags.Meta{
-				Name: "remote",
+				Name: flags.FlagRemote,
 				Usage: flags.Usage{
 					Description: "Specify the name or ID of a remote Realm app to export",
 				},
 			},
 		},
+		flags.BoolFlag{
+			Value: &cmd.inputs.IncludeNodeModules,
+			Meta: flags.Meta{
+				Name: flags.FlagIncludeNodeModules,
+				Usage: flags.Usage{
+					Description: "Export and include Realm app dependencies from an archive file",
+					Note:        "The allowed formats are as a directory or compressed into a .zip, .tar, .tar.gz, or .tgz file",
+				},
+			},
+		},
+		flags.BoolFlag{
+			Value: &cmd.inputs.IncludePackageJSON,
+			Meta: flags.Meta{
+				Name: flags.FlagIncludePackageJSON,
+				Usage: flags.Usage{
+					Description: "Include Realm app dependencies in the diff from a package.json file",
+				},
+			},
+		},
+		// TODO(REALMC-10088): Remove this flag in realm-cli 3.x
 		flags.BoolFlag{
 			Value: &cmd.inputs.IncludeDependencies,
 			Meta: flags.Meta{
-				Name:      "include-dependencies",
+				Name:      flags.FlagIncludeDependencies,
 				Shorthand: "d",
 				Usage: flags.Usage{
-					Description: "Export and include Realm app dependencies",
+					Description: "Export and include Realm app dependencies from an archive file",
+					Note:        "The allowed formats are as a directory or compressed into a .zip, .tar, .tar.gz, or .tgz file",
 				},
+				Deprecator: flags.Forwarded{To: flags.FlagIncludeNodeModules},
 			},
 		},
 		flags.BoolFlag{
 			Value: &cmd.inputs.IncludeHosting,
 			Meta: flags.Meta{
-				Name:      "include-hosting",
+				Name:      flags.FlagIncludeHosting,
```

Comment:
same thing here, no need for the `const` (but if we did need it, lets keep it scoped to `commands/app/flags.go`)